usaspending_api/awards/serializers.py

```diff
@@ -1,6 +1,6 @@
 from rest_framework import serializers
 
-from usaspending_api.accounts.serializers import TasSerializer
+from usaspending_api.accounts.serializers import SubmissionAttributesSerializer, TasSerializer
```

Comment:
Why not import from `usaspending_api.submissions.serializers`?

Reply:
Didn't realize there was one in there. I'll compare the 2 and see which one is needed. If they're identical, I'll use `usaspending_api.submissions.serializers`